test/group-helper.test.js

```diff
@@ -0,0 +1,52 @@
+const { parse, sep } = require('path');
```

Comment:
make this under `test/lib/my/path/as/it/is/in/lib`

Reply:
@evenstensberg I kept in `test/lib/utils/` in the latest commit. Please check if this is fine !